lint_rules/lint_rules/ifs_coding_standards_2011.py

```diff
@@ -486,20 +495,31 @@ def check_subroutine(cls, subroutine, rule_report, config, **kwargs):
             root_expr_map = defaultdict(list)
             for expr in expr_list:
                 expr_root = FindExpressionRoot(expr).visit(node)[0]
```

Comment:
This could go inside the conditional

Reply:
Done!

Reply:
Can't see the change, did this go under somehow? The entire loop could be wrapped in the conditional:
```
if node.source and node.source.string:
    for expr in expr_list:
        expr_root = FindExpressionRoot(expr).visit(node)[0]
        root_expr_map[expr_root] += [expr]
```

Reply:
I'd piggybacked some other stuff onto this PR but separated that into #126. This change accidentally wound up in that PR. Are you happy to leave it as is or should I move that commit to this PR?

Reply:
Either is fine - but I can't find the change in #126 either?

Reply:
I think I misunderstood your original comment. I thought you meant move `_op_map` inside the conditional. But you were referring to moving the `for expr_root, exprs in root_expr.items()` loop into the `if node.source and node.source.string` conditional right?

Reply:
Right, sorry, yes - that's what I meant. It wasn't possible before because the `source` property was used from the expression but now it's loop-invariant.

Reply:
Right. Sure, I'll move that loop inside the conditional, and also rebase #126 👍